examples/with-supertokens/package.json

```diff
@@ -9,7 +9,7 @@
     "next": "latest",
     "react": "17.0.1",
     "react-dom": "17.0.1",
-    "supertokens-auth-react": "^0.15.0",
+    "supertokens-auth-react": "^0.16.0",
```

Comment:
It looks like this version was published under a different tag than latest, should we ensure a version on the latest tag is used here? 

<img width="1247" alt="Screen Shot 2021-10-01 at 09 17 46" src="https://user-images.githubusercontent.com/22380829/135635413-ef430c8a-0f2a-41d0-8f09-513d6ff71252.png">

Reply:
Oh right! I think there was an error in our scripts for publishing the SDK. We will add the latest tag to `0.16.0`. So feel free to merge this now if you'd like :)